# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp3/MP3Demuxer.h
# Commit: 763b34de3c14
# Full Hash: 763b34de3c143cc9777be616395f1e9d95583240
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-20 21:19:23
# Regressor Bug: 1843003
# File Overlap Count: 1
# Description:
#   Bug 1843003 - Keep track of how much padding has been stripped, and don't trim more than necessary. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183736
# ==============================================================================

diff -r 81c600b19271 -r 763b34de3c14 dom/media/mp3/MP3Demuxer.h
--- a/dom/media/mp3/MP3Demuxer.h	Thu Jul 20 13:17:58 2023 +0000
+++ b/dom/media/mp3/MP3Demuxer.h	Thu Jul 20 13:17:58 2023 +0000
@@ -177,6 +177,7 @@
   uint32_t mEncoderDelay = 0;
   // Number of frames to skip at the end
   uint32_t mEncoderPadding = 0;
+  int32_t mRemainingEncoderPadding = 0;
   // End of stream has been found
   bool mEOS = false;
 };
